burn/Cargo.toml

```diff
@@ -53,6 +53,9 @@ candle = ["burn-core/candle"]
 # Experimental
 experimental-named-tensor = ["burn-core/experimental-named-tensor"]
 
+# Records
+record-serde = ["burn-core/record-serde"]
```

Comment:
Do you think it would be cleaner to rename to `module-adapter`? `record-serde` isn't very clear.

Reply:
`record-item-custom-serde` maybe? module-adapter is way off from the intention. This feature provides a custom serializer/deserializer for record-items.